packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -117,6 +159,9 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 			__experimentalCanUserUseUnfilteredHTML: canUseUnfilteredHTML,
 			__experimentalUndo: undo,
 			outlineMode: hasTemplate,
+			__experimentalCreateEntity: createEntity,
```

Comment:
Exposing this on the editor settings allow us to access it within the format library without creating a dependency on WordPress Core APIs.

Reply:
`__experimentalCreateEntity` suggests it may create any entity, but it's only for creating pages. What do you think about `__experimentalCreatePage`?